packages/docusaurus-plugin-openapi/src/openapi/openapi.ts

```diff
@@ -213,10 +213,12 @@ export async function readOpenapiFiles(
     );
 
     // TODO: Add config for inlcude/ignore
-    const sources = await Globby(["**/*.{json,yaml,yml}"], {
-      cwd: openapiPath,
-      ignore: GlobExcludeDefault,
-    });
+    const sources = (
+      await Globby(["**/*.{json,yaml,yml}"], {
+        cwd: openapiPath,
+        ignore: GlobExcludeDefault,
+      })
+    ).filter((x) => !x.includes("_category_")); // todo: regex exclude?
```

Comment:
I think `_category_` should be excluded by `GlobExcludeDefault` above, which excludes anything that starts with an underscore?

Reply:
![image](https://user-images.githubusercontent.com/10419222/146711535-8e296bb1-3d2c-4572-8b12-f9cd50e145bf.png)

doesn't appear to work

Reply:
Ah yea looks like it doesn't ignore json files with `_`
```ts
  // Ignore files starting with _
  '**/_*.{js,jsx,ts,tsx,md,mdx}',

  // Ignore folders starting with _ (including folder content)
  '**/_*/**',

  // Ignore tests
  '**/*.test.{js,jsx,ts,tsx}',
  '**/__tests__/**',
```

Reply:
maybe try something like this then:
```ts
[...GlobExcludeDefault, '**/_*.{json,yaml,yml}']
```